### PR TITLE
Add Code of Conduct link + Release Management note

### PR DESCRIPTION
**Pull request by CescaGarrett**
Added link to new Code of Conduct and flagged that medic-docs does not involve release management and acceptance testing.

### CODE REVIEW

**Comment by garethbowen** [comment]
@CescaGarrett Sorry this slipped through the cracks. Can you please look at resolving the conflict and then assign me as a reviewer and I'll get it merged?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -85,6 +86,17 @@ If you believe you've found a security vulnerability in one of our products or p
...
+
+**Note:** medic-docs does not involve release management and acceptance testing. Help us maintain the quality of our documentation by submitting a pull request (PR) with any suggested changes. 
+
+Is our documentation up to date? Have we covered everything we should? Could our wording be improved? Read our [Documentation Style Guide](https://github.com/medic/medic-docs/blob/master/development/docs-style-guide.md) then open a pull request with your suggested changes or additions.
+Want to talk about Documentation generally? Join our [Community Forum](https://communityhealthtoolkit.org/forum)!
+
+### Code of Conduct
+
+All maintainers and contributors are required to act according to our [Code of Conduct](https://github.com/medic/medic/blob/master/CODE_OF_CONDUCT.md). Thank you for your help building a positive community and a safe environment for everyone.
+=======
```
Comment (garethbowen) [inline CONTRIBUTING.md]: It looks like this line was left in by mistake.

Comment (CescaGarrett) [reply CONTRIBUTING.md]: So sorry, which line? My eyes are crossing from editing decks today!

Comment (garethbowen) [reply CONTRIBUTING.md]: @CescaGarrett Sorry for the late response - I missed your comment somehow. The `=====` line looks like it was left in from the merge conflict resolution.

Comment (garethbowen) [reply CONTRIBUTING.md]: @CescaGarrett Did you see my comment? I'd love to get this merged - let me know if I can help.

Comment (CescaGarrett) [reply CONTRIBUTING.md]: Done!

**Pull request merged**